apps/ledger-live-mobile/src/components/RootNavigator/types/SendFundsNavigator.ts

```diff
@@ -9,7 +9,10 @@ import type {
   Transaction,
   TransactionStatus,
 } from "@ledgerhq/live-common/generated/types";
-import type { Transaction as EthereumTransaction } from "@ledgerhq/live-common/families/ethereum/types";
+import type {
+  Transaction as EthereumTransaction,
+  TransactionRaw as EtherumTransactionRaw,
+} from "@ledgerhq/live-common/families/ethereum/types";
```

Comment:
❌  importing ethereum specific from a generic component

maybe it means you need these to be moved to families folder & exported through "generated"?